packages/@tailwindcss-upgrade/src/template/codemods/legacy-classes.test.ts

```diff
@@ -39,3 +39,33 @@ test.each([
 
   expect(await legacyClasses(designSystem, {}, candidate)).toEqual(result)
 })
+
+test('does not match false positives', async () => {
```

Comment:
This test name feels a bit weird. If it doesn't match them they're no longer false positives. I can't immediately think of a better name tho